common/common.C

```diff
@@ -62,7 +62,7 @@ void export_mtx(const word fieldName, std::shared_ptr<const gko::LinOp> A,
                 const word matrixFormat)
 {
     std::string folder{db.time().timePath()};
-    std::filesystem::create_directories(folder);
+    //std::filesystem::create_directories(folder);
```

Comment:
```suggestion
    std::filesystem::create_directories(folder);
```
For the integration tests to work we still need this. To fix the issue you can use 
`target_link_libraries(OGL PUBLIC Ginkgo::ginkgo stdc++fs)` in the root CMakeLists.txt.